platform-sdk/swirlds-state-impl/src/main/java/com/swirlds/state/merkle/disk/OnDiskReadableKVState.java

```diff
@@ -83,24 +88,56 @@ protected V readFromDataSource(@NonNull K key) {
     @Override
     protected Iterator<K> iterateFromDataSource() {
         // Log to transaction state log, what was iterated
-        logMapIterate(getStateKey(), virtualMap, keyCodec);
-        return new OnDiskIterator<>(virtualMap, keyCodec);
+        logMapIterate(getLabel(), megaMap, keyCodec);
+        return new OnDiskIterator<>(megaMap, keyCodec);
     }
 
     /** {@inheritDoc} */
     @Override
     public long size() {
-        final var size = virtualMap.size();
+        final var size = megaMap.size();
```

Comment:
Please, mark this method as deprecated (or even drop it completely, if possible). State sizes will be tracked using a different mechanism.

Reply:
I chose to mark this method as deprecated rather than remove it, as it's still in use, and removing it would further increase the size of this PR.